CuratedContent/KanbanAndScrumMakingTheBestOfBoth.md

```diff
@@ -0,0 +1,61 @@
+They do so in a way that provides deeper insight into these process frameworks than can be gained by just studying them in isolation.
+The authors break down Scrum and Kanban into a set of tools and process building blocks that can be used to construct custom processes for individual projects.
+Kanban is likely the least prescriptive framework for lean/agile project/development management that has even been proposed that is based up with solid theory and evidence.
+By incrementally adding some tools from Scrum, Kanban can be extended and modified to address almost any project and create a continuum of lean/agile project management processes from basic Kanban to full-blow Scrum.
+
+Part I of this mini-book is about 50 pages and provides an excellent overview and discussion of Scrum and Kanban and breaks them down into their pieces to come up with a powerful toolbox of building-blocks.
+This material is a fast read and provides the basic terminology for discussion of Kanban, Scrum and agile methods in general.
+Part II of this mini-book is a 50-page case study in the application of these building blocks and experience from a real project.
+Unlike extended examples in some books, the case study presented in Part II really enhances the material covered in Part I.
+
+This mini-book is packed with helpful diagrams and cartoons that convey the concepts being discussed in a very power way.
+Just the diagrams and cartoons alone make this mini-book worth the time to browse and is a great way to refresh these concepts in one's mind.
+
+Considering there are much longer books on lean and agile methods, this mini-book, in my view, provides the best usage of one's time in to get a grip on these concepts with the least amount of investment.
+I can't recommend this mini-book more strongly if one is interested in improving one's software project management processes using lead/agile methods.
+
+The ideas and insights of this mini-book were used as the foundation for the software development management processes in the CSE Consortium For Advanced Simulation Of Light Water Reactors (CASL) project and especially in the Nuclear Reactor Physics Integration Focus area  starting in 2010.
+By using the iterative retrospective process and with experiments, a very successful iteration-based Kanban process was constructed and variations of that process endure in some of the groups that participated in the CASL project after the main 10-year project ended.
+
+At the time of this writing, this mini-book is available for free download from [infoq.com](https://www.infoq.com/mini-books/kanban-scrum-mini-book/) but you must register with infoq.com to get it.
+A printed version of the mini-book that can be mailed to you is also available for a very small fee.
+
+<img src='https://github.com/betterscientificsoftware/images/raw/master/KanbanAndScrumMakingTheBestOfBoth.png' class='logo' />
+
+
+<!---
```

Comment:
We only have 4 meta-tags right now, others are deprecated or reserved. List of metadata tags can be found: https://betterscientificsoftware.github.io/bssw.io/bssw_content_metadata.html

Reply:
It would be good to remove the deprecated tags in all of the existing `*.md` files since I (and likely other people) use these as a template for new articles.

Reply:
> It would be good to remove the deprecated tags in all of the existing `*.md` files since I (and likely other people) use these as a template for new articles.

We havent had resources to do that yet. So, we only changed the examples filed mentioned on the EB docs website

Reply:
I agree that the cleanup would be nice, but it is not a priority, since deprecated tags are ignored.